dev/bots/codesign.dart

```diff
@@ -0,0 +1,56 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io';
+import 'package:path/path.dart' as path;
+
+class CodesignFailure {
+  const CodesignFailure({
+    this.binaryPath,
+    this.exitCode,
+    this.stderr,
+  });
+
+  final int exitCode;
+  final String binaryPath;
+  final String stderr;
+}
+
+void main() {
+  // TODO(fujino): parse codesigning manifest for all files to verify
+  final List<String> binaries = <String>[
+    path.join('bin', 'cache', 'dart-sdk', 'bin', 'dart'),
+    path.join('bin', 'cache', 'artifacts', 'engine', 'darwin-x64', 'flutter_tester'),
+    path.join('bin', 'cache', 'artifacts', 'engine', 'darwin-x64', 'gen_snapshot'),
```

Comment:
font-subset?

Do we need to verify the ios-deploy and imobiledevice are signed asd well?

Reply:
I was gonna (in a future PR) have this script parse the actual manifest file of all binaries in the engine to sign, but now I realize that that won't be easily accessible from the framework. I will hard-code them all here.

Reply:
Talked offline - it should be possible to just find all the binaries in bin/cache with the right MIME type and check those.